web/modules/custom/usagov_benefit_category_search/usagov_benefit_category_search.module

```diff
@@ -4,7 +4,9 @@ use Drupal\usagov_benefit_category_search\Form\BenefitCategorySearchForm;
 
 function usagov_benefit_category_search_preprocess_node(&$vars) {
   if ($vars['page'] === TRUE && $vars['view_mode'] === 'full' && $vars['node']->getType() === 'basic_page') {
-    if (\Drupal::service('path.matcher')->isFrontPage()) {
+    if (\Drupal::service('path.matcher')->isFrontPage()
+        || in_array(\Drupal::service('path.current')->getPath(), ['/node/123', '/node/519']) // English and Spanish Government Benefits pages
```

Comment:
Tempting as it is, I don't think we want to rely on these node IDs. I'd rather we checked for content in the Benefits Callout field(s).

Reply:
Do we want some other way of preventing the callout from showing on other navigation pages like `/es/inmigracion-ciudadania-estados-unidos`? I wasn't sure about hard coding paths but if its enough to tell content admins "only fill these in if you want it to display" ...

Then I realized, I think I set up the conditional fields so that callout only shows when you edit the benefits pages. Content editors shouldn't see them on other navigation pages and this check is redundant.